docs/StardustDocs/topics/createDataFrame.md

```diff
@@ -153,6 +153,12 @@ df.add("length") { value.length }
 
 Creates a [`DataFrame`](DataFrame.md) from [`Iterable<T>`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-iterable/) with one column:
 "columnName: `DataColumn<T>`".
+An easy way to create a [`DataFrame`](DataFrame.md) when you have a list of File, URL or a structure
```

Comment:
...from an Iterable...

This is an easy way...

..list of `Files`, `URLs`, or any kind of structure you want...

In a notebook, ... see the number of rows...

..with the parts of the data you're...

It could be the `File`'s content, a specific section of an `HTML` document, some metadata, etc.